backend/Squirrel.Core/Squirrel.Core.BLL/MappingProfiles/UserRegisterProfile.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoMapper;
+using Google.Apis.Auth;
+using Squirrel.Core.Common.DTO.Auth;
+using Squirrel.Core.DAL.Entities;
+
+namespace Squirrel.Core.BLL.MappingProfiles;
+
+public sealed class UserRegisterProfile : Profile
+{
+    public UserRegisterProfile()
+    {
+        CreateMap<User, UserRegisterDto>()!.ReverseMap();
+        CreateMap<GoogleJsonWebSignature.Payload, UserRegisterDto>()
+            .ForMember(m => m.Email, s => s.MapFrom(f => f.Email))
```

Comment:
It seems that if the source and destination properties have the same name, AutoMapper will automatically map them by convention even if the mapping is not explicitly defined

Reply:
yes, I know this, but just wanted to show explicitly what properties I map. Ok, I'll remove it